knative-operator/pkg/common/serving.go

```diff
@@ -12,8 +12,18 @@ import (
 
 var log = Log
 
+const (
+	// defaultDomainTemplate is a value for domainTemplate in config-network.
+	// As Knative on OpnShift uses OpenShift's wildcard cert the domain level must have "<sub>.domain", not "<sub1>.<sub2>.domain".
```

Comment:
nit: OpnShift -> OpenShift but don't hold up merge for this unless other changes are needed